Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -781,15 +774,17 @@ namespace OpenInfraPlatform {
 						if (!convert3DPointsTo2D(boundID, plane, loopVertices2D, loopVertices3D, faceLoopReversed)) 
 						{
 							conversionFailed = true;
-							BLUE_LOG(warning) << "#" << faceID << "= IfcFace: loop could not be projected";
-							continue;
+							std::stringstream ExceptionMessage;
+							ExceptionMessage << "#" << face->getId() << "= IfcFace: loop could not be projected";
+							throw oip::InconsistentGeometryException(face, ExceptionMessage.str().c_str());
 						}
 
 						if (loopVertices2D.size() < 3) 
 						{
 							conversionFailed = true;
-							BLUE_LOG(warning) << "#" << faceID << "= IfcFace: path_loop.size() < 3";
-							continue;
+							std::stringstream ExceptionMessage;
+							ExceptionMessage << "#" << face->getId() << "= IfcFace: path_loop.size() < 3";
+							throw oip::InconsistentGeometryException(face, ExceptionMessage.str().c_str());

```

Comment:
```suggestion
							throw oip::InconsistentGeometryException(face, "loopVertices2D.size() < 3");
```
Perhaps a more descriptive error would be even better.

Reply:
Resolved:  d142297